canosp2020/preprocessing.py

```diff
@@ -76,20 +77,33 @@ def apply_spacy_docs(inputs):
     return df
 
 
+def apply_num2words(inputs):
```

Comment:
I suggest using multiprocessing if this things take more than 30sec to run on our entire corpus

Otherwise looks good to me.